compiler/rustc_middle/src/query/mod.rs

```diff
@@ -972,7 +972,6 @@ rustc_queries! {
         key: ty::ParamEnvAnd<'tcx, GlobalId<'tcx>>
     ) -> EvalToValTreeResult<'tcx> {
         desc { "evaluate type-level constant" }
-        remap_env_constness
```

Comment:
Why was this removed?

Reply:
If I remember correctly we got a problem with the `constness` of a ParamEnv. Using `remap_env_constness` in some circumstances removes the `constness` of the ParamEnv I think.